Ntuplizer/plugins/MuonsNtuplizer.cc

```diff
@@ -198,12 +211,16 @@ void MuonsNtuplizer::fillBranches( edm::Event const & event, const edm::EventSet
     nBranches_->mu_isGlobalMuon.push_back(mu.isGlobalMuon());  
     nBranches_->mu_isTrackerMuon.push_back(mu.isTrackerMuon());  
     nBranches_->mu_isMediumMuon.push_back(isMediumMuon(mu));
+    nBranches_->mu_isTrackerHighPtMuon.push_back(isTrackerHighPtMuon(mu, &*firstGoodVertex));
    
       
     double normChi2	   = -99;
     int    trackerHits     = -99;
     int    pixelHits	   = -99;
     int    globalMuonHits  = -99;
+    double innerTrack_pt = -99;
```

Comment:
Hi Alberto, 

It seems this variable is not used at all. 
Could you remove in the next round of iteration?

Yuta